PWGDQ/DataModel/ReducedInfoTables.h

```diff
@@ -239,7 +239,7 @@ DECLARE_SOA_TABLE(ReducedMuonsExtra, "AOD", "RTMUONEXTRA", //!
                   fwdtrack::NClusters, fwdtrack::PDca, fwdtrack::RAtAbsorberEnd,
                   fwdtrack::Chi2, fwdtrack::Chi2MatchMCHMID, fwdtrack::Chi2MatchMCHMFT,
                   fwdtrack::MatchScoreMCHMFT, fwdtrack::MatchMFTTrackID, fwdtrack::MatchMCHTrackID,
-                  fwdtrack::MCHBitMap);
+                  fwdtrack::MCHBitMap, fwdtrack::MIDBitMap, fwdtrack::MIDBoards);
```

Comment:
Similar to my comment on https://github.com/AliceO2Group/AliceO2/pull/7033, you could add dynamic columns to decode the board numbers

Reply:
Done in the new commit. thanks.